acme/acme/messages.py

```diff
@@ -596,12 +592,6 @@ class Authorization(ResourceBody):
     def challenges(value: List[Dict[str, Any]]) -> Tuple[ChallengeBody, ...]:  # type: ignore[misc]  # pylint: disable=no-self-argument,missing-function-docstring
         return tuple(ChallengeBody.from_json(chall) for chall in value)
 
-    @property
-    def resolved_combinations(self) -> Tuple[Tuple[ChallengeBody, ...], ...]:
```

Comment:
This removal is probably technically going to want a warning.

Reply:
I have created #9365 to track new things we want to deprecate in the next release, including `resolved_combinations`.